apps/emqx_bridge_mongodb/test/emqx_bridge_mongodb_SUITE.erl

```diff
@@ -290,13 +312,17 @@ create_bridge(Config, Overrides) ->
 delete_bridge(Config) ->
     Type = mongo_type_bin(?config(mongo_type, Config)),
     Name = ?config(mongo_name, Config),
-    emqx_bridge:remove(Type, Name).
+    emqx_bridge:check_deps_and_remove(Type, Name, [connector, rule_actions]).
```

Comment:
I solved this in a different way in my PostgreSQL PR. I updated `emqx_bridge:remove` so that it would delete both the action and connector (should be fine since `emqx_bridge:remove` should only be called from test cases).

Reply:
I see, I thought it's this way by intention. Don't think it hurts to leave it like this.